src/runtime_src/doc/toc/build.rst

```diff
@@ -19,40 +19,47 @@ Build the runtime
 
 ::
 
-   cd XRT/build
+   cd build
    ./build.sh
 
 ``build.sh`` script builds for both Debug and Release profiles.  On RHEL/CentOS, if ``build.sh`` was accidentally run prior to enabling the devtoolset, then it is necessary to clean stale files makefiles by running ``build.sh clean`` prior to the next build.
 
 Build RPM package on RHEL/CentOS or DEB package on Ubuntu
 .........................................................
 
-::
+The package is actually automatically built for the ``Release``
+version but not for the ``Debug`` version::
 
-   cd XRT/build/Release
+   cd build/Release
+   make package
+   cd ../Debug
    make package
 
 Install the XRT RPM package
 ...........................
 
-::
+Install with (the actual package name might differ) ::
 
    yum reinstall ./XRT-2.1.0-Linux.rpm
 
 Install the XRT DEB package
 ...........................
 
-::
+Install with (the actual package name might differ) ::
 
-   apt install --reinstall ./XRT-2.1.0-Linux.deb
+   sudo apt install --reinstall ./xrt_201830.2.1.0_18.10.deb
```

Comment:
Add sudo for the yum command also.  I'd prefer to  leave out the 201830 and 18.10 from package name; these change depending on release and OS

Reply:
Yes this is why I have added `(the actual package name might differ)` in the line above, so I think it can remain like that. Before it was wrong in the sense that almost none of the characters would happen like that. So it is at least a plausible example. :-)

I can add `sudo` for the `yum` command too. I did not tried the RedHat style, so I did not update this part, but yes I can add it.

The philosophy behind the changes was to make only minor changes since I do not have the big picture of the project. I am expecting some other newcomers to do improvements on the RedHat side for example, now it is open-source ! :-)